cart-service/src/utils/logger.utils.ts

```diff
@@ -10,6 +18,26 @@ export const getLogger = () => {
     loggerInstance = createApplicationLogger({
       level: process.env.LOG_LEVEL || 'info',
     });
+
+    if (
+      process.env.OTEL_EXPORTER_OTLP_ENDPOINT &&
+      process.env.OTEL_EXPORTER_OTLP_ENDPOINT_API_KEY
+    ) {
+      const loggerProvider = new LoggerProvider();
+      const logExporter = new OTLPLogExporter({
+        url: `${process.env.OTEL_EXPORTER_OTLP_ENDPOINT}/v1/logs`,
```

Comment:
It's probably worth pulling the OTEL settings from the configuration object rather than accessing the environment variables directly here (to keep things consistent across the codebase)

Reply:
Yep - cool. That's resolved.